flink-tests/src/test/java/org/apache/flink/test/streaming/runtime/BigUserProgramJobSubmitITCase.java

```diff
@@ -122,8 +83,9 @@ public String map(Integer value) throws Exception {
 		}).addSink(resultSink);
 
 		JobGraph jobGraph = StreamingJobGraphGenerator.createJobGraph(env.getStreamGraph());
-		CLIENT.setDetached(false);
-		CLIENT.submitJob(jobGraph, BigUserProgramJobSubmitITCase.class.getClassLoader());
+		final ClusterClient<?> clusterClient = MINI_CLUSTER_RESOURCE.getClusterClient();
+		clusterClient.setDetached(false);
+		clusterClient.submitJob(jobGraph, BigUserProgramJobSubmitITCase.class.getClassLoader());
```

Comment:
You're changing the behavior of this test since it no longer uses a REST client. This should be reverted.

Reply:
Arg, this is a very good catch! Thanks a lot. Will fix it.